### DISCUSSION TITLE
[Bug] "Print PDF" make LateX equation very light

### DISCUSSION THREAD

**Discussion by RedHeadEmile** [docmost/docmost#1605] [Ideas]
Let me show an example.
When on the Docmost UI :
![image](https://github.com/user-attachments/assets/ce9ee97b-d027-44c1-a2ed-d79cb916ce4e)
![image](https://github.com/user-attachments/assets/56b2ff80-8c4e-485d-abd0-f561a4cc5553)

After "Print PDF" :
![image](https://github.com/user-attachments/assets/178d5786-e7ea-415a-a488-1ef2e5aa4b78)
![image](https://github.com/user-attachments/assets/d9ea7213-8810-47e8-8245-f01c26438978)

The line are very light and they are hard to see. Why aren't they solid black like in the Docmost UI ?

**Comment by krishanjangid** [comment]
due to font support it i guess. there are bunch of pdf libraries they can use to print as it is visible in Docmost UI.
